### ISSUE TITLE
Resource ciscoise_active_directory_join_domain throws error i.e. Failure when executing JoinDomain

### ISSUE THREAD

**Issue by prerna-prernas** [CiscoISE/terraform-provider-ciscoise#78]
**Prerequisites**
* [ yes ] Have you tested the operation in the API directly?
* [ yes ] Do you have the latest Terraform provider version?
* [ yes ] Review the [compatibility matrix](https://github.com/CiscoISE/terraform-provider-ciscoise/tree/main#compatibility-matrix) before opening an issue. 

**Describe the bug**
Resource ciscoise_active_directory_join_domain is not getting created and error out when performing JoinDomain operation. Observed that same operation is also not working using API.
<img width="687" alt="Screenshot 2023-01-25 at 10 46 03 AM" src="https://user-images.githubusercontent.com/123530369/214485709-98d0e43f-03d4-4573-8795-ccc317013fe1.png">
``` 
## Create Active Directory Join Point

resource "ciscoise_active_directory" "test_ad" {
  provider = ciscoise
  parameters {
    ad_scopes_names = "Default_Scope"
    adgroups {}
    description = "testing-tf"
    domain      = "${local.domain}"
    name        = "tf-test-ad"
  }
}

## Join the node to the AD Domain

resource "ciscoise_active_directory_join_domain" "test_join" {
  provider = ciscoise
  lifecycle {
    create_before_destroy = true
  }
  parameters {
    id = ciscoise_active_directory.test_ad.parameters[0].id
    additional_data {
      name  = "username"
      value = "${local.ad_username}"
    }
    additional_data {
      name  = "password"
      value = "${local.ad_password}"
    }
    additional_data {
      name  = "node"
      value = "${local.node}"
    }
    additional_data {
      name  = "orgunit"
      value = "${local.org_unit}"
    }
  }
}
```

**Expected behavior**
Expecting to spin the resource and perform the mapped operation i.e. to join AD with node specified in additional_data parameter defined in resource.

**Screenshots**
NA

**Environment (please complete the following information):**
* ISE version and patch: 3.1 and 5
* Terraform version: v1.2.5
* ISE provider version: 0.6.11-beta
* OS version: macOS Monterey v12.6.1

**Additional context**
NA

**Comment by prerna-prernas** [comment]
Additionally observed that the same resource works fine if don't supply "orgunit" as a parameter. But then if as per requirement if we need to supply it then how to achieve the same ?

`resource "ciscoise_active_directory" "test_ad" {
  provider = ciscoise
  parameters {
    ad_scopes_names = "Default_Scope"
    adgroups {}
    description = "testing-tf"
    domain      = "${local.domain}"
    name        = "tf-test-ad"
  }
}

resource "ciscoise_active_directory_join_domain" "test_join" {
  provider = ciscoise
  lifecycle {
    create_before_destroy = true
  }
  parameters {
    id = ciscoise_active_directory.test_ad.parameters[0].id
    additional_data {
      name  = "username"
      value = "${local.ad_username}"
    }
    additional_data {
      name  = "password"
      value = "${local.ad_password}"
    }
    additional_data {
      name  = "node"
      value = "${local.node}"
    }
  }
}`

**Comment by grg1bbs** [comment]
This is related to the following open bug for the API:
https://bst.cloudapps.cisco.com/bugsearch/bug/CSCwe13780

**Comment by fmunozmiranda** [comment]
Hey @1homas, do you have more info about this issue?

**Comment by grg1bbs** [comment]
This API bug is currently expected to be fixed in ISE 3.1 patch 7 (possibly May timeframe, but could change without notice)

**Comment by grg1bbs** [comment]
@fmunozmiranda, I believe you can close this issue. The API bug has been fixed in 3.1 patch 7 and 3.2 patch 2
